Add extended version of output_display.py script

HeathenUK opened the pull request:
Based on the display_multiplex sketch, to create an image for those of us that don't program EEPROM with Arduino.

HeathenUK committed:
Add extended version of output_display.py script

Based on the display_multiplex sketch, to create an image for those of us that don't program EEPROM with Arduino.

vascofazza reviewed (changes requested):
Thank you for the PR!
Could you please improve on the existing MK1_CPU/code/output_display.py ?
It contains most of the logic and it would just resolve in code duplication.

HeathenUK committed:
Add XOR function to standard library

I'm sure this can be optimised significantly, but here's a readable XOR function to complete the set

HeathenUK commented:
There's something not quite right on the negative two's complement values, will revisit this.
